client/src/api/taskAPI.js

```diff
@@ -1,7 +1,7 @@
 import network from "../network";
 
-export const getStudentTasksAPI = (config) =>
-	network.get(`/Task/StudentTasks`, config);
+export const getStudentTasksAPI = () => network.get("/Task/StudentTasks");
```

Comment:
Don't remove config param from this func.
Check this line:
https://github.com/Vlad1408EZ/SDLCSimulator/blob/dima/addTeacherView/client/src/slices/tasksSlice.js#:~:text=dispatch(setTaskSearchValue(value))%3B-,getStudentTasksAPI,-(%7B